tests/_fixtures/metronome/job.json

```diff
@@ -111,7 +111,7 @@
         }
       ]
     }
-  ] ,
```

Comment:
i wonder if we should ignore this change – not a biggie but cypress keeps changing these fixtures every time it runs and it's pretty annoying...